exasol_script_languages_container_tool/cli/commands/deploy.py

```diff
@@ -0,0 +1,166 @@
+    build_options,
+)
+from exasol_integration_test_docker_environment.cli.options.docker_repository_options import (
+    docker_repository_options,
+)
+from exasol_integration_test_docker_environment.cli.options.system_options import (
+    luigi_logging_options,
+    system_options,
+)
+from exasol_integration_test_docker_environment.cli.termination_handler import (
+    TerminationHandler,
+)
+from exasol_integration_test_docker_environment.lib.api.common import add_options
+
+from exasol_script_languages_container_tool.cli.options.flavor_options import (
+    flavor_options,
+)
+from exasol_script_languages_container_tool.cli.options.goal_options import (
+    release_options,
+)
+from exasol_script_languages_container_tool.lib import api
+
+# This text will be displayed instead of the actual value, if found in an environment
+# variable, in a prompt.
+SECRET_DISPLAY = "***"
+
+
+class SecretParams(Enum):
+    """
+    This enum serves as a definition of confidential parameters which values should not be
+    displayed in the console, unless the user types them in the command line.
+
+    The enum name is also the name of the environment variable where the correspondent
+    secret value can be stored.
+
+    The enum value is also the name of the cli parameter.
+    """
+
+    DB_PASSWORD = "db-pass"
+    BUCKETFS_PASSWORD = "bucketfs-password"
+
+
+def secret_callback(ctx: click.Context, param: click.Option, value: Any):
+    """
+    Here we try to get the secret parameter value from an environment variable.
+    The reason for doing this in the callback instead of using a callable default is
+    that we don't want the default to be displayed in the prompt. There seems to
+    be no way of altering this behaviour.
+    """
+    if value == SECRET_DISPLAY:
+        secret_param = SecretParams(param.opts[0][2:])
+        return os.environ.get(secret_param.name)
+    return value
+
+
+@cli.command(short_help="Deploys the script-language-container in the database.")
+@add_options(flavor_options)
+@click.option("--bucketfs-host", type=str, required=True)
+@click.option("--bucketfs-port", type=int, required=True)
+@click.option("--bucketfs-user", type=str, required=True)
+@click.option("--bucketfs-name", type=str, required=True)
+@click.option("--bucket", type=str, required=True)
+@click.option("--bucketfs-use-https", type=bool, default=False)
+@click.option(
+    f"--{SecretParams.BUCKETFS_PASSWORD.value}",
+    type=str,
+    prompt="BucketFS password",
+    prompt_required=False,
+    hide_input=True,
+    default=SECRET_DISPLAY,
+    callback=secret_callback,
+)
+@click.option("--path-in-bucket", type=str, required=False, default="")
+@add_options(release_options)  # 1
+@click.option("--release-name", type=str, default=None)
+@add_options(build_options)  # 7
+@add_options(docker_repository_options)  # 4
+@add_options(system_options)  # 2
+@add_options(luigi_logging_options)  # 2
+@click.option("--ssl-cert-path", type=str, default="")
```

Comment:
I think, these should move upwards to the bucketfs parameters

Reply:
Done